specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2025-01-01-preview/mfe.json

```diff
@@ -9,6 +9,90 @@
     "https"
   ],
   "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.MachineLearningServices/availableQuota/default": {
+      "get": {
+        "tags": [
+          "Quota"
+        ],
+        "summary": "Get available MaaS PTU quota.",
+        "operationId": "PTUQuota_GetAvailable",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/AvailableQuota"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get available MaaS PTU quota.": {
+            "$ref": "./examples/PTUQuota/getAvailable.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.MachineLearningServices/quotaAndUsage": {
```

Comment:
>/subscriptions/{subscriptionId}/providers/Microsoft.MachineLearningServices/quotaAndUsage [](http://example.com/codeflow?start=5&length=89)

You need to add the corresponding singleton/point GET API as well for getting individual items. This is required for core ARM features.